lib/toPaths.js

```diff
@@ -245,4 +245,4 @@ function isSafeNumber(val) {
 }
 
 // export for testing
-module.exports._isSafeNumber = isSafeNumber;
+module.exports.isSafeNumber = isSafeNumber;
```

Comment:
Let's not change this internal API since falcor-transformer currently depends on it.

Reply:
I've added it back, but i believe we should make a new PR to falcor-transformer which uses isIntegerKey instead of _isSafeNumber throughout, and then remove _isSafeNumber entirely.

Do you foresee any issues with that? Do any client teams currently use _isSafeNumber?